scrapyd_k8s/config.py

```diff
@@ -21,13 +21,27 @@ def launcher_cls(self):
         pkg, cls = repo.rsplit('.', 1)
         return getattr(import_module(pkg), cls)
 
+    def joblogs(self):
+        if self._config.has_section('joblogs'):
+            return self._config['joblogs']
+        else:
+            return {}
+
+    def joblogs_provider_args(self, provider):
+        if not self._config.has_section('joblogs.%s' % provider):
+            return None
+        return self._config['joblogs.%s' % provider]
+
     def listprojects(self):
         return self._projects
 
     def project(self, project):
         if project in self._projects:
             return ProjectConfig(self._config, project, self._config['project.' + project])
 
+    def get_namespace(self):
+        return self.scrapyd().get('namespace', 'default')
```

Comment:
We don't use the prefix `get_` elsewhere, so I think we can just call this method `namespace`.